whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapService.java

```diff
@@ -245,6 +250,38 @@ public Response help(@Context final HttpServletRequest request) {
                 .build();
     }
 
+    @GET
+    @Produces({MediaType.APPLICATION_JSON, CONTENT_TYPE_RDAP_JSON})
+    @Path("/{objectType}/rirSearch1/{relation}/{key:.*}")
+    public Response relationSearch(
+            @Context final HttpServletRequest request,
+            @PathParam("objectType") RdapRequestType requestType,
+            @PathParam("relation") RelationType relationType,
+            @PathParam("key") final String key,
+            @QueryParam("status") String status) {
+
+        //TODO: [MH] Status is being ignored until administrative resources are included in RDAP. If status is not
+        // given or status is inactive...include administrative resources in the output. However, if status is active
+        // return just non administrative resources, as we are doing now.
+        if (status != null && (relationType.equals(RelationType.DOWN) || relationType.equals(RelationType.BOTTOM))){
```

Comment:
use stringUtils isEmpty method to check for empty string as well

Reply:
true, I always forget about the stringUtils...

Reply:
done